.ci/deploy/devfile-registry.yaml

```diff
@@ -69,6 +69,11 @@ objects:
             limits:
               cpu: 100m
               memory: ${DEVFILE_INDEX_MEMORY_LIMIT}
+          env:
+          - name: ENABLE_TELEMETRY
+            value: true
+          - name: REGISTRY
```

Comment:
What's this environment variable for exactly? I thought we were only going to use one project in segment?

And we should use a more descriptive environment variable name.

Reply:
It should also be configurable with a parameter

Reply:
Yes, only one project for segment and woopra, this is the registry name that is used as identifier for telemetry to know  which registry the data come from. I will update the description.